framework/i18n/Formatter.php

```diff
@@ -787,6 +787,50 @@ public function asRelativeTime($value, $referenceTime = null)
         }
     }
 
+    /**
+     * Formats the value as a time interval.
+     * @param mixed $value the value to be formatted, in ISO8601 format.
+     * @return string the formatted result.
+     */
+    public function asInterval($value)
```

Comment:
Maybe, it's better to rename method to `asDuration`? Interval is more about relative time